pkg/scheduler/framework/types.go

```diff
@@ -720,10 +722,23 @@ func max(a, b int64) int64 {
 
 // resourceRequest = max(sum(podSpec.Containers), podSpec.InitContainers) + overHead
 func calculateResource(pod *v1.Pod) (res Resource, non0CPU int64, non0Mem int64) {
+	inPlacePodVerticalScalingEnabled := utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling)
 	resPtr := &res
 	for _, c := range pod.Spec.Containers {
-		resPtr.Add(c.Resources.Requests)
-		non0CPUReq, non0MemReq := schedutil.GetNonzeroRequests(&c.Resources.Requests)
+		var non0CPUReq, non0MemReq int64
```

Comment:
why are we defining them here? just add `:=` when assigning the values below just like the original code.

Reply:
fixed.

Reply:
fixed and reviewed. resolving